src/main/java/codeu/controller/ConversationServlet.java

```diff
@@ -60,7 +60,7 @@ void setUserStore(UserStore userStore) {
    * for use by the test framework or the servlet's init() function.
    */
   void setConversationStore(ConversationStore conversationStore) {
-    this.conversationStore = conversationStore;
```

Comment:
would revert this change since you're just adding a space.

Reply:
How do I revert a change?

Reply:
Just remove the space.